pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```
@@ -230,6 +231,9 @@ qtModule {
     [Paths]
     Prefix = ..
     EOF
+  '' + lib.optionalString (lib.versions.majorMinor qtCompatVersion == "5.15") ''
+    # Fix for out-of-sync QtWebEngine and Qt releases (since 5.15.3)
+    sed 's/${version} /${qtCompatVersion} /' -i "$dev"/lib/cmake/*/*Config.cmake
```

```suggestion
    sed 's/${version} /${qtCompatVersion} /' -i "$dev"/lib/cmake/*/*Config.cmake
```

This will replace `5.15.3` with `5.15`, is that intended?

---

No, `qtCompatVersion` equals `qtbase.src.version`, which is `5.15.2`.